PsychSourceGL/Cohorts/FTGLTextRenderer/libptbdrawtext_ftgl.cpp

```diff
@@ -450,11 +449,7 @@ void PsychSetTextBGColor(int context, double* color)
 // renderbackend needs a different geometric setup:
 void PsychSetTextViewPort(int context, double xs, double ys, double w, double h)
```

Comment:
Ok, I see. It is dangerous (and annoying) to have something from which you expect a certain effect, only to find out later it doesn't do what it advertises. We don't know how this might be used later.

In that case, my projection matrix would be:
`gluOrtho2D(_vxs, _vxs + _vw, _vys + _vh, _vys);`
i.e., reversing the third and fourth argument so that it has an origin in top-left, consistent with the rest of PTB. Ok?